index.js

```diff
@@ -48,6 +48,13 @@ module.exports.cacheStores = {
   MemoryCacheStore: require('./lib/cache/memory-cache-store')
 }
 
+try {
+  const SqliteCacheStore = require('./lib/cache/sqlite-cache-store')
+  module.exports.cacheStores.SqliteCacheStore = SqliteCacheStore
+} catch (_) {
+  // Do nothing
```

Comment:
I think we should check for expect error

Reply:
Wdym?

Reply:
Basically verify that this errors with a certain error code, and if not rethrow the error